message.go

```diff
@@ -58,6 +64,34 @@ func DescribeMessage(m Message) string {
 	}
 }
 
+// ValidatableMessage is a message that can validate itself.
+//
+// This interface can be implemented to fine-grained validation of messages.
```

Comment:
```suggestion
// This interface can be implemented to perform fine-grained validation of messages.
```

Reply:
I can't believe Kev, our resident profession proof reader missed this 😱